test/test_http_get_collect.py

```diff
@@ -43,11 +61,28 @@ def test_http_get_collect_async_false(sess):
     response = sess.execute(
         text(
             """
-        select * from net.http_collect_response(:request_id, async:=False); 
+        select * from net.http_collect_response(:request_id, async:=True);
```

Comment:
`async:=true`  - perhaps there is a reason for the uppercase though?

Reply:
python muscle memory